### DISCUSSION TITLE
Good SvelteKit template

### DISCUSSION THREAD

**Discussion by jm61** [pocketbase/pocketbase#618] [Show and tell]
Hello to all, 
here is a [great template from Dana Woodman](https://github.com/danawoodman/sveltekit-auth-example) for SK and PocketBase, enjoy

**Comment by jkdoshi** [comment]
I just wrote one - https://github.com/spinspire/pocketbase-sveltekit-starter

It works for me. But it's very new. So please give it a try and provide feedback in issues and discussions for the project.

It has the following features:

- A fully static frontend with SvelteKit and adapter-static
- Demo of "posts" and file uploads
- Demo of login
- Demo of customized Go binary using PocketBase "as a framework". Includes custom endpoint `/api/hello` and event hooks.
- Demo of containerization using Dockerfile
- Serving your frontend and your backend using the single PocketBase binary
- config-driven hooks: execute a command/script or a webhook POST when records change in a table
